src/myAds/myAdsReducer.js

```diff
@@ -99,11 +99,8 @@ export function toQuery(search) {
     return query;
 }
 
-function* getMyAds(action) {
+function* getMyAds() {
     try {
-        if (action.type !== CHANGE_MY_ADS_PAGE) {
-            yield put({ type: RESET_MY_ADS_PAGE });
```

Comment:
Flytter resetPage til egen action for å ikke resette pagen hver gang man laster inn mine stillinger